tests/CMakeLists.txt

```diff
@@ -67,6 +66,7 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/light_validation_sync_test.py ${CMAKE
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/trace_plugin_test.py ${CMAKE_CURRENT_BINARY_DIR}/trace_plugin_test.py COPYONLY)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/nested_container_multi_index_test.py ${CMAKE_CURRENT_BINARY_DIR}/nested_container_multi_index_test.py COPYONLY)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/large-lib-test.py ${CMAKE_CURRENT_BINARY_DIR}/large-lib-test.py COPYONLY)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/launcher.py ${CMAKE_CURRENT_BINARY_DIR}/launcher.py COPYONLY)
```

Comment:
I noticed in an  file you stripped the superfluous  `CMAKE_CURRENT_SOURCE_DIR` and `CMAKE_CURRENT_BINARY_DIR` locations from `configure_file` ([docs](https://cmake.org/cmake/help/latest/command/configure_file.html)) - should we do that here?

Better yet, can we simplify this more?
```
  configure_file(nodeos_run_remote_test.py . COPYONLY)
```

Reply:
We typically try to avoid spurious diffs while adding features.  We should update every CMakeLists.txt which is unnecessarily verbose, but not in this PR.  New ones that are part of this PR, or were created for it, it's appropriate, and I'll make those changes, but the rest of the codebase will have to wait.